test/jdk/javax/management/remote/mandatory/connection/DefaultAgentFilterTest.java

```diff
@@ -91,6 +115,40 @@ public void op2(String s, HashSet<String> params) {
         public void op3(MyTestObject obj, String s, HashMap<String, String> param) {
             System.out.println("Invoked op3");
         }
+
+        public void setMyAttribute(boolean b) {
+            this.myAttribute = b;
+            System.out.println("Invoked setMyAttribute");
+        }
+
+        public boolean getMyAttribute() {
+            System.out.println("Invoked getMyAttribute");
+            return myAttribute;
+        }
+
+        public void setMyAttribute2(String s) {
+            String old = myAttribute2;
+            this.myAttribute2 = s;
+            Notification n = new AttributeChangeNotification(this, sequenceNumber++, System.currentTimeMillis(),
+                                                             "String attribute changed", "myAttribute2", "String",
+                                                             old, this.myAttribute2);
+            sendNotification(n);
+            System.out.println("Invoked setMyAttribute2");
+        }
+
+        public String getMyAttribute2() {
+            System.out.println("Invoked getMyAttribute2");
+            return myAttribute2;
+        }
+
+        @Override
+        public MBeanNotificationInfo[] getNotificationInfo() {
+            String[] types = new String[] { AttributeChangeNotification.ATTRIBUTE_CHANGE };
+            String name = AttributeChangeNotification.class.getName();
+            String description = "An attribute of this MBean has changed";
+            MBeanNotificationInfo info = new MBeanNotificationInfo(types, name, description);
+            return new MBeanNotificationInfo[] {info};
```

Comment:
Nit: It can be useful to print that this method was invoked the same as in other methods.

Reply:
got it